app/src/main/java/org/oppia/app/options/OptionsActivityPresenter.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.app.options
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.domain.profile.ProfileManagementController
+import org.oppia.util.logging.Logger
+import javax.inject.Inject
+
+/** The presenter for [OptionsActivity]. */
+@ActivityScope
+class OptionsActivityPresenter @Inject constructor(
+  private val activity: AppCompatActivity,
+  private val logger: Logger,
+  private val profileManagementController: ProfileManagementController) {
+
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.option_activity)
+    if (getOptionFragment() == null) {
+      activity.supportFragmentManager.beginTransaction().add(
+        R.id.option_fragment_placeholder,
+        OptionsFragment(activity,profileManagementController,logger)
```

Comment:
Spacing issues: ` OptionsFragment(activity, profileManagementController, logger)`

Reply:
Getting this warning in add function:

<img width="826" alt="Screenshot 2020-02-03 at 11 18 32 AM" src="https://user-images.githubusercontent.com/9396084/73628681-f8a6fd00-4676-11ea-9f41-aefea392d051.png">

Reply:
I checked this thing. I am not able to see any warning. Can you please check it once again.